privacyidea/cli/pitokenjanitor/utils/findtokens.py

```diff
@@ -745,3 +745,60 @@ def remove_tokeninfo(ctx, tokeninfo_key):
             token_obj.del_tokeninfo(tokeninfo_key)
             token_obj.save()
             click.echo(f"Removed tokeninfo '{tokeninfo_key}' for token {token_obj.token.serial}")
+
+
```

Comment:
Please also fix the typo in the `_get_tokenlist()` parameter `tokencontaner_filter` and all its occurrences.